modules/exploits/windows/imap/mercury_login.rb

```diff
@@ -4,87 +4,79 @@
 ##
 
 class MetasploitModule < Msf::Exploit::Remote
-  Rank = AverageRanking
+  Rank = NormalRanking
 
   include Msf::Exploit::Remote::Tcp
+  include Msf::Exploit::Remote::Seh
+  include Msf::Exploit::Remote::Egghunter
 
   def initialize(info = {})
     super(update_info(info,
-      'Name'           => 'Mercury/32 LOGIN Buffer Overflow',
-      'Description'    => %q{
+      'Name' => 'Mercury/32 less or equal than 4.01b LOGIN SEH Buffer Overflow',
+      'Description' => '
         This module exploits a stack buffer overflow in Mercury/32 <= 4.01b IMAPD
         LOGIN verb. By sending a specially crafted login command, a buffer
         is corrupted, and code execution is possible. This vulnerability was
         discovered by (mu-b at digit-labs.org).
-      },
-      'Author'         => [ 'MC' ],
+      ',
+
+      'Author'         => ['MC','Ivan Racic'],
```

Comment:
Generally the `Author` field makes use of comments to describe each author's contributions, such as:


```suggestion
      'Author'         => [
        'MC', # Discovery and Metasploit module
        'Ivan Racic' # Automatic targeting + egg hunter
      ],
```

Reply:
It looks like, for whatever reason, the original author (mu-b) was never credited appropriately as an author.

```ruby

      'Author'         =>
        [
          'mu-b', # Discovery
          'MC', # Metasploit
          'Ivan Racic' # Automatic targeting + egg hunter
        ],